lab4/lab4_task1.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8"/>
+    <title>lab4_1</title>
+    <style>
+        .sivtsev{
+        font-size: 5em;
+        color: #000;
+        text-shadow: 0 0 15px black;
```

Comment:
Text shadow is not the effect you need here. See, even the smallest letters are still perfectly visible on the shadow:
![image](https://user-images.githubusercontent.com/16776066/31585158-b9568f82-b1c5-11e7-9587-d2991f8bbf39.png)

You need them  to look more like this:
![image](https://user-images.githubusercontent.com/16776066/31585166-e39393c6-b1c5-11e7-8327-2aa2338a8e86.png)

Reply:
Check out https://css-tricks.com/almanac/properties/f/filter/

Reply:
got it